2021-06-25 12:02:06,799 - create_public - ERROR - Error detected
Traceback (most recent call last):
  File "C:\Users\Webdom16\PycharmProjects\TestsCovid\COVID_19_ 9880\Cоздание_исследований\create_public.py", line 37, in genlog
    self.create_public()
  File "C:\Users\Webdom16\PycharmProjects\TestsCovid\COVID_19_ 9880\Cоздание_исследований\create_public.py", line 85, in create_public
    driver.find_element_by_css_selector("#itemForm\:tabView\:homeAddressStreet_panel").click()
AttributeError: 'NoneType' object has no attribute 'click'
